packages/components/src/entities/SamplesTabbedGridPanel.tsx

```diff
@@ -309,6 +309,7 @@ export const SamplesTabbedGridPanel: FC<Props> = memo(props => {
     };
 
     const isMedia = activeModel?.queryInfo?.isMedia;
+    const isAllTab = (tabs.length > 1 || tabbedGridPanelProps?.alwaysShowTabs) && activeTabId === tabs[0];
```

Comment:
This seems a little presumptuous to say the first tab is always the All Samples tab. It seems better to use `isAllSamplesSchema()` to check if the current tab's model is indeed the all samples schema.

Reply:
done